### ISSUE TITLE
"Note off" notes don't work

### ISSUE THREAD

**Issue by vurpo** [sbechet/xmrsplayer#4]
When I add "note off" notes in my files, this library still keeps playing the previous notes like nothing happened. Currently I'm setting volume to 0 instead of using note off, as a workaround for this.

**Comment by vurpo** [comment]
[testcase.xm.zip](https://github.com/sbechet/xmrsplayer/files/14468712/testcase.xm.zip)

Here's a simple test XM file, which should have a note turning on and off periodically, but using xmrsplayer it's constantly on instead.

**Comment by sbechet** [comment]
Tested with ft2 clone v1.65: you right :(

**Comment by sbechet** [comment]
Can you test last version?

**Comment by vurpo** [comment]
I'll test later. I just released this demo: https://www.youtube.com/watch?v=V7GjQy8hxq4 / https://github.com/vurpo/laser-demo which uses xmrsplayer.

Although I ran into a bug here: on the demoparty's compo machine, the xmrsplayer ran way too fast (sample rate was correct, but it advanced through rows way too quickly). Didn't manage to investigate what the issue was, and it worked correctly on a different Windows and macOS machine I tried.
